_automating-configurations/api/create-workflow.md

```diff
@@ -72,12 +72,26 @@ PUT /_plugins/_flow_framework/workflow/<workflow_id>?update_fields=true
 You cannot specify both the `provision` and `update_fields` parameters at the same time.
 {: .note}
 
+If a workflow has been provisioned, you can update and reprovision the full template:
+
+```json
+PUT /_plugins/_flow_framework/workflow/<workflow_id>?reprovision=true
+{
+  <updated complete template>
+}
+```
+{% include copy-curl.html %}
+
+You can add new steps to the workflow but not delete them. Only steps for index settings, search pipelines, and ingest pipelines are currently supported for updating.
+{: .note}
```

Comment:
The {: .note} is not producing anything specific in markdown, can we remove?

Reply:
I think that renders on the actual documentation page, see the similar one above which renders like this:

![Screenshot 2024-09-04 at 4 51 44 PM](https://github.com/user-attachments/assets/a896df54-7b35-4b77-ba7d-10e5e9fbeeac)

Reply:
I see, I am okay with it then

Reply:
I anticipate we'll make this particular paragraph more detailed and link to a whole separate page (like the use case templates) for the remainder of the feature, so just trying to add the minimum this time.